### ISSUE TITLE
Give the Chargin' Targe the airblast resist (like a Mantreads) or afterburn immune

### ISSUE THREAD

**Issue by WillBMorra** [ValveSoftware/Source-1-Games#5622]
Nowadays almost noone using the Chargin' Targe because it has only resistences (not so good to be honest). So Tide Turner with a 75% recharge and movement and Splendid Screen with extra bash damage and faster charge cd. So why Chargin' Targe doesn't have any of this, because Pyro still OP agains Demo- and Demi(?)knights?

**Comment by m0rtykai** [comment]
i'm pretty sure this is for bug reports, not balance suggestions

**Comment by MeatwadMeatwad** [comment]
This is a terrible change. Demoknight needs weaknesses to be balanced.

**Comment by s01ch1r0** [comment]
Afterburn immunity could be reasonable, but considering it would give the danger shield to knight, it would be very unfair to dragon's fury pyros. It can be argued that DF not bypassing unlock-based burn immunity is a bug, though. 
Currently, degreaser afterburn is unaffected by damage resistance, and still serves to deny crit healing just as effectively,  as well as providing combo ability, with quick switching making it very hard for knights to fight degreaser pyros. So patching DF afterburn on DDS and then giving this shield that stat would be ideal.
I use targe more than other shields, and trust me, a knockback resistance as a base stat for it would not make it more useable. Surfing airblasts and explosions is fun, and knockback makes it so classes I resist damage from can keep me from rushing into their face.

**Comment by WillBMorra** [comment]
> This is a terrible change. Demoknight needs weaknesses to be balanced.

Agreed with a balance, but what I mean is not to give immune to airblast, but resist - so Pyro can cancel the charge, but can't permanently airblast you like a mile away repeatedly, more like slightly push away.

**Comment by WillBMorra** [comment]
> Afterburn immunity could be reasonable, but considering it would give the danger shield to knight, it would be very unfair to dragon's fury pyros. It can be argued that DF not bypassing unlock-based burn immunity is a bug, though. Currently, degreaser afterburn is unaffected by damage resistance, and still serves to deny crit healing just as effectively, as well as providing combo ability, with quick switching making it very hard for knights to fight degreaser pyros. So patching DF afterburn on DDS and then giving this shield that stat would be ideal. I use targe more than other shields, and trust me, a knockback resistance as a base stat for it would not make it more useable. Surfing airblasts and explosions is fun, and knockback makes it so classes I resist damage from can keep me from rushing into their face.

Yes, but i mean more like airblast resist than a immune - so Pyro still can cancell your charge, but now they airblast you not so far away

**Comment by sweglord227** [comment]
this doesn't make demoknight any stronger at all. it's just a massive "fuck you" to pyro players. also i don't think whomever dev is working on tf2 right now cares about balance changes, especially considering the onslaught of bug reports and crashes i've seen while scrolling through these.

**Comment by WillBMorra** [comment]
> this doesn't make demoknight any stronger at all. it's just a massive "fuck you" to pyro players. also i don't think whomever dev is working on tf2 right now cares about balance changes, especially considering the onslaught of bug reports and crashes i've seen while scrolling through these.

Fair point, but it's cool when something besides "nice" resistances because there's very solid choices through shields and I don't mind living the life of an arsonist a little...more difficult >:3 (not the case at all)
